internal/core/entities/scheduler.go

```diff
@@ -52,16 +53,19 @@ type Scheduler struct {
 	PortRange       *PortRange
 	CreatedAt       time.Time
 	MaxSurge        string `validate:"required,max_surge"`
+	Forwarders      []*forwarder.Forwarder
```

Comment:
@caiordjesus use Dive decorator to indicate that Forwarders[]'s will be validate when Scheduler is being validated

Reply:
Done! Thx